Makefile

```diff
@@ -349,7 +349,9 @@ install-chart: build/toolchain/bin/helm$(EXE_EXTENSION)
 		--set prometheus.enabled=false \
 		--set redis.enabled=true \
 		--set openmatch.monitoring.stackdriver.enabled=true \
-		--set openmatch.monitoring.stackdriver.gcpProjectId=$(GCP_PROJECT_ID)
+		--set openmatch.monitoring.stackdriver.gcpProjectId=$(GCP_PROJECT_ID) \
+		--set openmatch.e2eevaluator.install=true \
```

Comment:
I'd prefer not to add these to the default chart because this is the install.yaml file that customers will need to install the baseline Open Match.

I think the best approach is to install-chart first and then install-e2e-chart. You'll need to add install-e2e-chart in the cloudbuild.yaml file to deploy it. Use the deploy chart step to do this.